site/en/blog/deprecating-unload/index.md

```diff
@@ -0,0 +1,158 @@
+   reasons: [ "Internal Error", "Unload handler" ],
+   src: "",
+   url: "a.com"
+}
+```
+
+## Control access to unload
+
+Chrome will deprecate the `unload` event gradually. In the meantime you can use different tools to control this behavior and prepare for the upcoming deprecation. Keep in mind that you should not rely on these techniques in the long term and you should plan to migrate to the alternatives instead as soon as possible.
+
+The following options allow you to enable or disable `unload` handlers to test how your site would work without them so you can prepare for the upcoming deprecation. There are different types of policies:
+
+- [Permissions Policy](https://github.com/w3c/webappsec-permissions-policy/blob/main/permissions-policy-explainer.md): This is a platform API for site owners to control access to features, at a site or an individual page level, via the usage of HTTP headers.
+- [Enterprise policies](https://chromeenterprise.google/policies/): Tools for IT admins to configure Chrome for their organization or business. They can be configured via an admin panel, like the [Google Admin Console](https://support.google.com/a/answer/182076?hl=en).
+- [Chrome flags](/docs/web-platform/chrome-flags/): This allows an individual developer to change the unload deprecation setting to test impact on various sites.
+
+### Permissions Policy
+
+A Permissions Policy [has been added from Chrome 115](https://chromestatus.com/feature/5760325231050752) to allow sites to opt-out of using unload handlers and immediately benefit from bfcache usage to improve site performance. See [these examples on how to set this for your site](https://github.com/fergald/docs/blob/master/explainers/permissions-policy-unload.md#examples). This allows sites to get ahead of the unload deprecation.
+
+This [will be extended in Chrome 117](https://chromestatus.com/feature/5579556305502208) to allow sites to do the reverse, and to opt-in to continuing to try to fire unload handlers, as Chrome changes the default for these not to fire in future. See [these examples on how to continue to allow unload handlers to fire for your site](https://github.com/fergald/docs/blob/master/explainers/permissions-policy-deprecate-unload.md#reenabling-unload-for-a-frame). This opt-in will not remain forever and should be used to allow time for sites to migrate away from unload handlers.
+
+### Enterprise policy
+
+Enterprises that have software that depends on the `unload` event to function correctly can use the [`ForcePermissionPolicyUnloadDefaultEnabled` policy](https://chromium-review.googlesource.com/c/chromium/src/+/4730081) to prevent the gradual deprecation for devices under their control. By enabling this policy, the Permissions-Policy for unload will continue to default to enabled for all origins. A page may still set a stricter policy if it wants. Like the Permissions Policy opt-out, this is a tool to mitigate potential breaking changes but should not be used indefinitely.
```

Comment:
```suggestion
Enterprises that have software that depends on the `unload` event to function correctly can use the [`ForcePermissionPolicyUnloadDefaultEnabled` policy](https://chromium-review.googlesource.com/c/chromium/src/+/4730081) to prevent the gradual deprecation for devices under their control. By enabling this policy, the Permissions Policy for `unload` will continue to default to enabled for all origins. A page may still set a stricter policy if it wants. Like the Permissions Policy opt-out, this is a tool to mitigate potential breaking changes, but it should not be used indefinitely.
```